core/src/main/java/org/apache/accumulo/core/file/rfile/RFile.java

```diff
@@ -1555,5 +1561,42 @@ private void setInterruptFlagInternal(AtomicBoolean flag) {
     public void setCacheProvider(CacheProvider cacheProvider) {
       reader.setCacheProvider(cacheProvider);
     }
+
+    /**
+     * Returns an estimate of the number of entries that overlap the given extent. This is an
```

Comment:
Could move this javadoc up to FileSKVIterator.estimateEntries().  Its nice javadoc, putting it there would make it more widely visible.